server/flyway/local_sql/V1000__update_test_user_cognito_id.sql

```diff
@@ -5,6 +5,10 @@ UPDATE app_fam.fam_user
 SET cognito_user_id = 'test-idir_b5ecdb094dfb4149a6a8445a0mangled@idir'
 WHERE user_name = 'COGUSTAF';
 
+UPDATE app_fam.fam_user
+SET user_guid = 'b5ecdb094dfb4149a6a8445a0mangled'
+WHERE user_name = 'COGUSTAF';
```

Comment:
since now in the schema we make the requester's user_guid to be required, not optional anymore, add a fake user_guid for the test user